templates/conf.erb

```diff
@@ -108,8 +111,20 @@
     nosharedscripts
 <%  end -%>
     start <%= @start %>
+<%  if @size -%>
+    size <%= @size %>
+<%  end -%>
+<%  if @shred -%>
+    shred
+<%    if @shredcycles -%>
+    shredcycles <%= @shredcycles.to_s %>
+<%    end -%>
+<%  end -%>
+<%  if @su && @su_user && @su_group -%>
+    su <%= @su_user %> <%= @su_group %>
+<%  end -%>
 }
-<% if @tabooext -%>
+<% unless (@tabooext.nil? || @tabooext.empty?) -%>
```

Comment:
shouldn't the type for tabooext prevent it from being empty?

Reply:
Yes. will fix.